packages/lambda-tiler/src/tiler.ts

```diff
@@ -1,10 +1,9 @@
 import { Epsg, TileMatrixSet } from '@basemaps/geo';
 import { GoogleTms } from '@basemaps/geo/build/tms/google';
 import { Nztm2000Tms } from '@basemaps/geo/build/tms/nztm2000';
-import { Nztm2000AgolTms } from '@basemaps/shared/build/alternative.tms/nztm2000.agol';
 import { Tiler } from '@basemaps/tiler/build/tiler';
 
-export const DefaultTilers = [new Tiler(GoogleTms), new Tiler(Nztm2000Tms), new Tiler(Nztm2000AgolTms)];
+export const DefaultTilers = [new Tiler(GoogleTms), new Tiler(Nztm2000Tms)];
```

Comment:
` get(epsg: Epsg, alt?: string): Tiler | undefined {
        return Tilers.map.get(TileMatrixSet.getId(epsg, alt));
    },
`
This Tile.get and TileMatrixSet.getId can remove the alt parameter as well.

Reply:
Tile.get also gets removed in the next PR I think